components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -157,8 +241,29 @@ subroutine crm_physics_register()
    call pbuf_add_field('CRM_CLD_RAD', 'physpkg', dtype_r8, dims_crm_rad, idx)
    call pbuf_add_field('CRM_QRAD',    'global',  dtype_r8, dims_crm_rad, idx)
    
-   cldo_idx = pbuf_get_index('CLDO')
-   call pbuf_add_field('CLDO', 'global', dtype_r8, (/pcols ,pver, dyn_time_lvls/), cldo_idx  )
+   call pbuf_add_field('CLDO',  'global', dtype_r8, (/pcols, pver, dyn_time_lvls/), cldo_idx  )
+   call pbuf_add_field('CLD',   'global', dtype_r8, (/pcols, pver, dyn_time_lvls/), cld_idx)
+   call pbuf_add_field('CONCLD','global', dtype_r8, (/pcols, pver, dyn_time_lvls/), concld_idx)
```

Comment:
Does this field have any meaning without offline physics?

Reply:
I don't think so... but it looks like it's accessed in RRTMG, but then never used. So getting rid of it would create an error there, right? I could go delete those lines in the CRM version of RRTMG to fix this.

Reply:
Actually, it's used in COSP. While we don't let users call COSP right now with MMF, I think we should maybe leave it until I refactor the COSP interface to play nice with MMF.

Reply:
sounds good.